README.md

```diff
@@ -344,9 +344,9 @@ Because of the limitations of the CORS policy, macros icons must be stored in `/
 - [ ] Fix transition animations
 - [ ] Macros and Commands editor
 - [ ] Macros Menu categories
-- [ ] Localisation
+- [ ] Localization
```

Comment:
I don't think it's a typo, "Localisation" is the correct spelling in British English